.Rbuildignore

```diff
@@ -6,3 +6,4 @@ sonar.project.properties
 ^working_example.R
 
 notepad.R
```

Comment:
Do these need to be there? (I know this is not part of this PR, but sonarcloud is not used anymore, and those two .R files should also not be part of the package, so can we remove these too?)

Reply:
Let's remove them.

```suggestion
```